src/Board.js

```diff
@@ -5,73 +5,59 @@ import PropTypes from 'prop-types'
 const BOARD_SIZE = 10
 const BOARD_A_CHAR_CODE = 'A'.charCodeAt()
 const SHIP_SIZE = 3
-const MARKER_TYPE_EMPTY_CSS = 'none'
-const MARKER_TYPE_SHIP_CSS = 'ship'
-const MARKER_TYPE_HIT_CSS = 'hit'
-const MARKER_TYPE_MISS_CSS = 'miss'
-const ROW_LABELS = Array.from(Array(BOARD_SIZE + 1).keys()).splice(1)
-const COLUMN_LABELS = Array.from(Array(BOARD_SIZE).keys()).map((elem) =>
-	String.fromCharCode(elem + BOARD_A_CHAR_CODE)
-)
+const MARKER_TYPE_EMPTY = 'none'
+const MARKER_TYPE_SHIP = 'ship'
+const MARKER_TYPE_HIT = 'hit'
+const MARKER_TYPE_MISS = 'miss'
+const ROW_RANGE = Array.from(Array(BOARD_SIZE + 1).keys()).splice(1)
+const COLUMN_RANGE = Array.from(Array(BOARD_SIZE).keys())
+	.map(elem => String.fromCharCode(elem + BOARD_A_CHAR_CODE))
 
 function Board (props) {
 	const [grid, setGrid] = useState(initGrid())
 	const [clickedAdjSquares, setClickedAdjSquares] = useState([])
 	const [hasShipPlaced, setHasShipPlaced] = useState(false)
 	const [hasShipSunk, setHasShipSunk] = useState(false)
-
-	return (
-		<div className='board'>
-			<div className='row-label'>
-				{ROW_LABELS.map((elem) => (
-					<div key={elem}>{elem}</div>
-				))}
-			</div>
-			<div className='column-label'>
-				{COLUMN_LABELS.map((elem) => (
-					<div key={elem}>{elem}</div>
-				))}
-			</div>
-			<div className='grid'>{renderGrid()}</div>
-		</div>
-	)
+	const [invalidClick, setInvalidClick] = useState(false)
 
 	function initGrid () {
 		const grid = new Map()
 
-		ROW_LABELS.forEach((row) => {
-			COLUMN_LABELS.forEach((col) => {
-				grid.set(col + row, MARKER_TYPE_EMPTY_CSS)
+		ROW_RANGE.forEach(row => {
+			COLUMN_RANGE.forEach(col => {
+				grid.set(col + row, MARKER_TYPE_EMPTY)
```

Comment:
I would highly suggest that instead of using the addition here, to do a concatenation of strings
\``${col}${row}`\`
especially because this is javascript. It just makes it easier to know what I'm expecting to see.

Reply:
and please do this every where else, because I was seriously confused for an hour